Cargo.toml

```diff
@@ -76,3 +76,6 @@ sha1 = "0.10.6"
 tempfile = "3.10.1"
 tokio = { version = "1.36.0", features = ["full"] }
 tower = { version = "0.4.13", features = ["full"] }
+
+[patch.crates-io]
+"hyper" = { git = "https://github.com/apollographql/hyper.git", tag = "header-customizations-20241108" }
```

Comment:
We do this to unblock in the short term, but is there a path forward for having this functionality upstream?

Reply:
yes, PR to upstream https://github.com/hyperium/hyper/pull/3773